src/styles/components/_cell.scss

```diff
@@ -0,0 +1,60 @@
+.cell {
+  box-sizing: border-box;
+  width: $day-size;
+  height: $day-size;
+  position: relative;
+  background-color: $color-day-main;
+  border: $day-border;
+  transition: all 0.5s;
+
+  &:hover {
+    background-color: $color-day-hover;
+    transform: translateY(-20px);
+  }
+
+  @for $i from 1 through $num-days {
+    &--#{$i}::before {
+      content: "#{$i}";
+      position: absolute;
```

Comment:
There is not reason to use `position: absolute;`. Use text positioning and `line-height`